js/main.ts

```diff
@@ -27,7 +24,8 @@ function startMsg(cmdId: number): Uint8Array {
   return builder.asUint8Array();
 }
 
-window["denoMain"] = () => {
+/* tslint:disable-next-line:no-default-export */
+export default function denoMain() {
```

Comment:
Why not use "export function main() {" rather than "export default" ?

Reply:
I did it this way to make minimal changes.

What will happen then when the bundle iife is run is that that will be a named export of bundle.  Currently the bundle is named `denoMain` and so therefore the bootstrap function would move to `denoMain.main()` if we went that way.

If we expect the bundle to generate a global variable named `deno` and then export a function named `main`, we can change the bootstrap to call `deno.main()` instead of `denoMain()`.  Personally that makes a little bit more sense, and who knows in the future if there is other "real" code that needs to be put into the `deno` global object.